kuma/payments/jinja2/payments/terms.html

```diff
@@ -14,64 +14,10 @@
 {% endblock %}
 
 {% block document_head %}
-    <div class="contribution-page-header payment-confirmation">
-        <div class="column-container">
-            <img src="{{ static('img/hero-dino-blank.png') }}" class="backdrop-image" alt="" role="presentation" />
-            <div class="column">
-                <div class="column-half">
-                    <h1 class="highlight highlight-spanned">
-                        <span class="highlight-span">{{_('MDN Web Docs payment terms')}}</span>
-                    </h1>
-                </div>
-            </div>
-        </div>
-    </div>
+    {{ render_react("SPA", request.LANGUAGE_CODE, request.get_full_path(), True)|safe }}
```

Comment:
```suggestion
    {{ render_react("SPA", request.LANGUAGE_CODE, request.get_full_path(), {
        "contribution_support_email": settings.CONTRIBUTION_SUPPORT_EMAIL,
     })|safe }}
```

Note that in `jinja2/payments/index.html` we just used `email` for the key that we're passing. That was maybe a bad idea because it's too generic of a word. 
If you like this suggestion, can you refactor it in `jinja2/payments/index.html` too?